### PR TITLE
Update FAQ and public documentation to add more info on SignalingBadRequest related error codes

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:** Update FAQ and public documentation to add more info on SignalingBadRequest related error codes

**Testing**
`npm run build:release` and check the html files generated.

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
By verifying the html doc pages.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
No. It's a documentation change.

4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No.

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by vidya-mohan — APPROVED** [review]

File: guides/07_FAQs.md [hunk 1]
```diff
@@ -126,6 +126,26 @@ The Amazon Chime SDK does not support scheduling meetings ahead of time. The mom
...
 
+### What does it mean when Amazon Chime SDK JS throws error with status code `SignalingBadRequest`, `MeetingEnded` or `SignalingInternalServerError` while establishing a signaling connection to the Chime Servers.
+
+From JoinAndReceiveIndexTask.ts, the Chime SDK for JavaScript throws the `SignalingBadRequest` Error. This indicates that the client's request to establish a signaling connection to the Chime Server has failed.
+
+There are 3 cases that can happen here.
+
+1. If the Chime SDK for JavaScript receives a `event.closeCode === 4410` from Amazon Chime backend, that means that an attendee is trying to join a meeting that has already ended. In that case, the Chime SDK for JavaScript throws an error with status code `MeetingEnded`
```
Comment (simmkyu) [inline guides/07_FAQs.md:135]: - The close code 4410 from the Chime backend indicates that an attendee has attempted to join an already-ended meeting. The Chime SDK for JavaScript throws an error with the status code `MeetingEnded`.

File: guides/07_FAQs.md [hunk 2]
```diff
@@ -126,6 +126,26 @@ The Amazon Chime SDK does not support scheduling meetings ahead of time. The mom
...
+2. If the Chime SDK for JavaScript receives `event.closeCode >= 4500 && event.closeCode < 4600`, that indicates an Internal Server Error in the Amazon Chime backend and the Chime SDK for JavaScript throws an error with status code `SignalingInternalServerError`. This indicates there is an issue with the Chime Server that requires further investigation. (e.g a 5xx)
```
Comment (simmkyu) [inline guides/07_FAQs.md:136]: * The close code between 4500 and 4599 (inclusive) indicates an internal server error in the Amazon Chime backend. In this case, the Chime SDK for JavaScript throws an error with the status code `SignalingInternalServerError`. Please create a GitHub issue including the Chime SDK browser logs.

File: guides/07_FAQs.md [hunk 3]
```diff
@@ -126,6 +126,26 @@ The Amazon Chime SDK does not support scheduling meetings ahead of time. The mom
...
+3. In other cases like when an attendee is deleted (by calling the [DeleteAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteAttendee.html) API), and the client tries to join a meeting with the credentials of the deleted attendee, the Chime SDK for JavaScript throws an error with status code `SignalingBadRequest`.
```
Comment (simmkyu) [inline guides/07_FAQs.md:137]: * If you attempt to join a Chime SDK meeting using the deleted attendee's response, the Chime SDK for JavaScript throws an error with the status code `SignalingBadRequest`. Note that you or someone can delete an attendee in the [DeleteAttendee API](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteAttendee.html) action.

nit: Move this bullet to the top to match the order: `SignalingBadRequest`, `MeetingEnded` or `SignalingInternalServerError` in your description above.

File: guides/07_FAQs.md [hunk 4]
```diff
@@ -126,6 +126,26 @@ The Amazon Chime SDK does not support scheduling meetings ahead of time. The mom
...
+
+Here is an example of `SignalingBadRequest` error from an INFO level browser log.
```
Comment (simmkyu) [inline guides/07_FAQs.md:139]: The INFO-level browser logs may include the following messages:

**Review by simmkyu — CHANGES_REQUESTED** [review]

**Comment by simmkyu** [comment]
I will create a PR incorporating those feedback. Thanks Anuran for adding!

**Pull request merged**